protoc-gen-elm/main.go

```diff
@@ -246,6 +246,9 @@ func processFile(inFile *descriptor.FileDescriptorProto) (*plugin.CodeGeneratorR
 		}
 	}
 
+	fg.P("")
```

Comment:
Probably move this to the top of the file (after the imports) rather than the bottom?